app/forms.py

```diff
@@ -0,0 +1,96 @@
+# import re
+from django import forms
+# from PIL import Image
+from django.contrib.auth.models import User
+from django.utils.translation import ugettext_lazy as _
+# from django.core.exceptions import ValidationError
+import datetime
+from django.forms.extras.widgets import SelectDateWidget
+
+
+class RegistrationForm(forms.Form):
+    this_year = datetime.date.today().year
```

Comment:
Since this is a class level attribute I think it's only set at load time.
That means that if an app is started in `2015` and is never restarted in `2016` it will never show `2016`.
I think we should set this value in line in line 36 instead of setting a class attribute.